docs/developer-notes.md

```diff
@@ -76,6 +76,49 @@ Scuri uses [Jasmine's spy feature](https://jasmine.github.io/2.0/introduction.ht
 We loosely follow the [Jasmine Style Guide](https://github.com/CareMessagePlatform/jasmine-styleguide) when writing tests. 
 One main guideline is that a `describe` block should be created for each method / scenario under test, and an `it` block should be created for each property being verified.
 
+# E2E Testing
+
+## Running E2E Tests
+
+E2E Tests can be executed by using `npm run e2e` which conducts testing  using the default `protractor.conf.js` in the Chrome Browser.   
+The following additional parameters would allow for further customisation,
+
+| Additional Parameter | Description | Full Command Example |
+| :---: | :-----: | :-------: |
+| `--protractor-config=e2e/protractor.*.conf.js` | Allows substitution of the default configuration file | `npm run e2e -- --protractor-config=e2e/protractor.firefox.conf.js` |
```

Comment:
```suggestion
| `--protractor-config=e2e/protractor.*.conf.js` | Allows substitution of the default configuration file | `npm run e2e -- --protractor-config=e2e/protractor.firefox.conf.js` (runs the tests using firefox) |
```

Let's briefly explain what the example does?

Reply:
![image](https://user-images.githubusercontent.com/39243082/110196039-c9b41980-7e7c-11eb-939d-aa265f6651b4.png)
Like this?

Reply:
Yes